scripts/shared/lib/clusters_kind

```diff
@@ -264,11 +259,47 @@ function download_ovnk() {
     )
 }
 
+function prepare_ovn_ic() {
```

Comment:
I'm not sure I understand what's the necessity of this function..

It doesn't seem to do anything special when building the image from source, why can't we use the published image in this case?

Or am I missing something here?

Reply:
Added docs inline, as discussed :+1:

Reply:
I think we can build he image both for ovn-ic and regular use cases as stale images are causing issues when kind image version is changed .
 https://github.com/submariner-io/shipyard/pull/1297#pullrequestreview-1578548108